src/daft-scan/src/anonymous.rs

```diff
@@ -33,7 +33,7 @@ impl AnonymousScanOperator {
 
 impl Display for AnonymousScanOperator {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{:#?}", self)
+        write!(f, "AnonymousScanOperator: File paths=[{}], Format-specific config = {:?}, Storage config = {:?}", self.files.join(", "), self.file_format_config, self.storage_config)
```

Comment:
We may want to do the same for `GlobScanOperator` as well! https://github.com/Eventual-Inc/Daft/blob/b6d6669582c35552a4aa757dbad2263c49ce4266/src/daft-scan/src/glob.rs#L217

Reply:
Yeah was thinking that, can do that in a follow up!